docs/book/src/reference/versions.md

```diff
@@ -63,53 +63,54 @@ These diagrams show the relationships between components in a Cluster API releas
 
 #### Core Provider (`cluster-api-controller`)
 
-|                   | CAPI v0.3 (v1alpha3) | CAPI v0.4 (v1alpha4) | CAPI v1.0 (v1beta1) | CAPI v1.1+v1.2 (v1beta1) |
-|-------------------|----------------------|----------------------|---------------------|--------------------------|
-| Kubernetes v1.16  | ✓                    |                      |                     |                          |
-| Kubernetes v1.17  | ✓                    |                      |                     |                          |
-| Kubernetes v1.18  | ✓                    | ✓ (only workload)    | ✓ (only workload)   | ✓ (only workload)        |
-| Kubernetes v1.19  | ✓                    | ✓                    | ✓                   | ✓ (only workload v1.2)   |
-| Kubernetes v1.20  | ✓                    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.21  | ✓                    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.22  | ✓ (only workload)    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.23* |                      | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.24  |                      |                      |                     | ✓                        |
+|                   | CAPI v0.3 (v1alpha3) | CAPI v0.4 (v1alpha4) | CAPI v1.0 (v1beta1) | CAPI v1.1+v1.2+v1.3 (v1beta1) |
```

Comment:
A few nits:

- Drop the "CAPI" prefix to all of these columns
- Split each column into a single version, without bundling versions, even if the matrix is siilar